client/src/Components/Sidebar/Sidebar.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import routes from "../../routes/routes";
+import "./Sidebar.scss";
+
+const Sidebar = ({ categories }) => {
+    return (
+        <div className="sidebar" aria-label="sidebar with categories list">
+            {
```

Comment:
sidebar could be wrapped inside html5 semantic - aside

Reply:
yeah, changes done